readme.md

```diff
@@ -8,69 +8,36 @@ Package your electron app in OS executables (.app, .exe, etc) via JS or CLI. Sup
 
 [![Build Status](https://travis-ci.org/maxogden/electron-packager.svg?branch=master)](https://travis-ci.org/maxogden/electron-packager)
 
-### installation
+## Installation
 
-```
+```sh
 # for use in npm scripts
-npm i electron-packager --save-dev
+npm install electron-packager --save-dev
 
 # for use from cli
-npm i electron-packager -g
+npm install electron-packager -g
 ```
 
-### usage
+## Usage
 
-```
-Usage: electron-packager <sourcedir> <appname> --platform=<platform> --arch=<arch> --version=<version>
-
-Required options
-
-platform           all, or one or more of: linux, win32, darwin (comma-delimited if multiple)
-arch               all, ia32, x64
-version            see https://github.com/atom/electron/releases
-
-Example            electron-packager ./ FooBar --platform=darwin --arch=x64 --version=0.25.1
-
-Optional options
-
-all                equivalent to --platform=all --arch=all
-out                the dir to put the app into at the end. defaults to current working dir
-icon               the icon file to use as the icon for the app. Note: Format depends on OS.
-app-bundle-id      bundle identifier to use in the app plist
-app-version        release version to set for the app
-build-version      build version to set for the app (OS X only)
-cache              directory of cached electron downloads. Defaults to '$HOME/.electron'
-helper-bundle-id   bundle identifier to use in the app helper plist
-ignore             do not copy files into App whose filenames regex .match this string
-prune              runs `npm prune --production` on the app
-overwrite          if output directory for a platform already exists, replaces it rather than skipping it
-asar               packages the source code within your app into an archive
-asar-unpack        unpacks the files to app.asar.unpacked directory whose filenames regex .match this string
-sign               should contain the identity to be used when running `codesign` (OS X only)
-version-string     should contain a hash of the application metadata to be embedded into the executable (Windows only).
-                   These can be specified on the command line via dot notation,
-                   e.g. --version-string.CompanyName="Company Inc." --version-string.ProductName="Product"
-                   Keys supported:
-                   - CompanyName
-                   - LegalCopyright
-                   - FileDescription
-                   - OriginalFilename
-                   - FileVersion
-                   - ProductVersion
-                   - ProductName
-                   - InternalName
+### From the Command Line
 
+Running electron-packager from the command line has this basic form:
 
 ```
+electron-packager <sourcedir> <appname> --platform=<platform> --arch=<arch> --version=<Electron version> [optional flags...]
+```
 
```

Comment:
:+1: I think what I want to do is a sub-section with an example command, which would also explain exactly what would happen. Output directory name, abbreviated tree, etc. Make it painfully obvious.